packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -78,6 +78,7 @@ export function deepMerge(target: any, source: any, isExtensionIncluded: boolean
     } else {
       // Directly assign values
       if (key === "__proto__" || key === "constructor") continue;
+      if (source[key] === "unknown" && combined[key] !== "unknown") continue;
```

Comment:
should we maybe define a list of variations of `unknown` and check for all of them in this step? i.e. something like this:
```
const unknownValues = ["unknown", "unk", "no known"];
if (unknownValues.some(unk => source[key].toLowerCase().includes(unk))) {
  ...
```